Allow env variable for fernet key

manelpb opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]
	

<!--- Before submitting a pull request please check the following -->

<!---  If this is a fix for a typo in code or documentation in the README please file an issue and let us sort it out we do not need a PR  --> 
<!---  Ask yourself if this modification is something the whole userbase will benefit from, if this is a specific change for corner case functionality or plugins please look at making a Docker Mod or local script  https://blog.linuxserver.io/2019/09/14/customizing-our-containers/ -->
<!---  That if the PR is addressing an existing issue include, closes #<issue number> , in the body of the PR commit message   -->
<!---  You have included links to any files / patches etc your PR may be using in the body of the PR commit message -->
<!--- We maintain a changelog of major revisions to the container at the end of readme-vars.yml in the root of this repository, please add your changes there if appropriate -->


<!--- Coding guidelines: -->
<!--- 1. Installed packages in the Dockerfiles should be in alphabetical order -->
<!--- 2. Changes to Dockerfile should be replicated in Dockerfile.armhf and Dockerfile.aarch64 if applicable -->
<!--- 3. Indentation style (tabs vs 4 spaces vs 1 space) should match the rest of the document -->
<!--- 4. Readme is auto generated from readme-vars.yml, make your changes there -->

------------------------------

We welcome all PR’s though this doesn’t guarantee it will be accepted.

## Description:
add option to send env variable for fernet key

## Benefits of this PR and context:
If running in kubernetes, we need to have the encryption key stored somewhere, or every time the deployment restarts, it will invalidate previous authentications

## How Has This Been Tested?
Tested with and without env variable. Without env variable, the current behaviour continues (generating the key).

aptalca commented:
@manelpb Thanks for the PR, I just added a check for validity of the env var and updated the readme

manelpb commented:
> @manelpb Thanks for the PR, I just added a check for validity of the env var and updated the readme

thank you! can we merge and release new docker images?

aptalca commented:
🤦‍♂️there is no openssl in the image, I'll switch it to python
